src/common/include/sirf/common/GeometricalInfo.h

```diff
@@ -90,11 +90,12 @@ class VoxelisedGeometricalInfo :
 	virtual bool operator==(const GeometricalInfo<num_dimensions, num_dimensions>& gi) const
 	{
 		const VoxelisedGeometricalInfo& vgi = (const VoxelisedGeometricalInfo&)gi;
+		float eps = 1e-4;
```

Comment:
```suggestion
		float eps = 1e-4;
```
```suggestion
		float eps = 1e-2;
```
Offsets etc are in mm, testing for 0.1 micrometer accuracy is far too much. Even 0.01mm is asking for a lot, but I guess we can see how that goes.

Reply:
ok